crates/hyperswitch_domain_models/src/business_profile.rs

```diff
@@ -216,6 +220,9 @@ pub enum ProfileUpdate {
     NetworkTokenizationUpdate {
         is_network_tokenization_enabled: bool,
     },
+    ClickToPayUpdate {
```

Comment:
where are we using this? is this necessary?

Reply:
what if, we want to update just clickToPay from dashboard!

Reply:
I do not see an endpoint that uses this update struct